### PR TITLE
fix: 🐛 handle NFT locked error in nft.canTransfer

### PR DESCRIPTION
**Pull request by polymath-eric**
### Description

adds support for a new error type for NFT canTransfer

### JIRA Link

[DA-1104](https://polymesh.atlassian.net/browse/DA-1104)

### Checklist

- [ ] Updated the Readme.md (if required) ?

[DA-1104]: https://polymesh.atlassian.net/browse/DA-1104?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/polkadot/augment-api-tx.ts [hunk 1]
```diff
@@ -105,8 +105,8 @@ import type {
   PolymeshPrimitivesTicker,
   PolymeshPrimitivesTransferComplianceTransferCondition,
   PolymeshPrimitivesTransferComplianceTransferConditionExemptKey,
-  PolymeshRuntimeDevelopRuntimeOriginCaller,
```
Comment (polymath-eric) [inline src/polkadot/augment-api-tx.ts:108]: There is no release with these types yet, so I generated against testnet

**Review by prashantasdeveloper — APPROVED** [review]

**Pull request merged**